LayoutTests/fast/forms/number/number-stepup-stepdown.html

```diff
@@ -54,204 +54,61 @@
 
 debug('Number type');
 input.type = 'number';
-
-debug('');
 debug('Invalid value');
-shouldBe('stepUp("", null, null)', '"1"');
-shouldBe('stepDown("", null, null)', '"-1"');
-
-debug('');
+shouldBeEqualToString('stepUp("", null, null)', '1');
+shouldBeEqualToString('stepDown("", null, null)', '-1');
 debug('Non-number arguments');
 shouldBe('stepUp("0", null, null, "0")', '"0"');
 shouldBe('stepDown("0", null, null, "0")', '"0"');
 shouldBe('stepUp("0", null, null, "foo")', '"0"');
 shouldBe('stepDown("0", null, null, "foo")', '"0"');
 shouldBe('stepUp("0", null, null, null)', '"0"');
 shouldBe('stepDown("0", null, null, null)', '"0"');
-
-debug('');
 debug('Normal cases');
 shouldBe('stepUp("0", null, null)', '"1"');
 shouldBe('stepUp("1", null, null, 2)', '"3"');
 shouldBe('stepUp("3", null, null, -1)', '"2"');
 shouldBe('stepDown("2", null, null)', '"1"');
 shouldBe('stepDown("1", null, null, 2)', '"-1"');
 shouldBe('stepDown("-1", null, null, -1)', '"0"');
-
-debug('');
-debug('Fractional cases')
```

Comment:
These are below deleted tests are same as in `number-stepup-stepdown-from-renderer.html`.

Since they are duplicated - I am deleting them from here.